src/components/Footer/Footer.js

```diff
@@ -231,7 +144,6 @@ const Footer = ({
           </View>
         </TouchableOpacity>
       </View>
-      <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} />
     </View>
   );
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BottomSheetMenu` component has been removed. If this component is no longer needed, this change is fine. However, if it's still required, you'll need to reintroduce this code.
```diff
-      <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} />
+      </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->